discovery-provider/src/app.py

```diff
@@ -450,6 +451,10 @@ def configure_celery(flask_app, celery, test_config=None):
                 "task": "index_oracles",
                 "schedule": timedelta(minutes=5),
             },
+            "index_rewards_manager": {
+                "task": "index_rewards_manager",
+                "schedule": timedelta(minutes=1),
```

Comment:
@hareeshnagaraj what should I set the timedelta to?
Is there a way to calculate optimal time?

Reply:
Set to 5 seconds